cfme/utils/ssh.py

```diff
@@ -202,7 +205,18 @@ def connect(self, hostname=None, **kwargs):
             self._connect_kwargs.update(kwargs)
```

Comment:
**REQUIRED:** The logic here I think could be clearer

```python
        if self.is_pod:
            logger.info("logging into openshift")
            login_cmd = 'oc login --username={u} --password={p}'
            login_cmd = login_cmd.format(u=self.oc_username,
                                         p=self.oc_password)
            self.run_command(command=login_cmd, ensure_host=True)
            return

        self._connect_kwargs.update(kwargs)
        self._check_port()
        # Only install ssh keys if they aren't installed (or currently being installed)
        return super(SSHClient, self).connect(**self._connect_kwargs)
```

Reply:
updated, please check